app/assets/stylesheets/components/common/_global_nav.scss

```diff
@@ -0,0 +1,359 @@
+// Global Navigation
+//
+// Styleguide Global Navigation
+
+$primary-background-colour: #fd5; // yellow;
+$secondary-background-colour: #fff; // white;
+$clump-height: 40;
+
+@mixin transition_mobile {
+  transition-property: transform;
+  transition-duration: 1s;
+}
+
+@mixin transition_desktop {
+  transition: none;
+}
+
+@mixin horizontal-separator {
+  border-bottom: solid 1px rgba(0, 0, 0, 0.375);
+  border-top: solid 1px rgba(0, 0, 0, 0.125);
+}
+
+.global-nav {
+  @include transition_mobile;
+
+  position: fixed;
+  z-index: 5;
+  width: 100%;
+  transform: translateX(100%);
+
+  @include respond-to($mq-l) {
+    @include transition_desktop;
+
+    position: relative;
+    transform: translateX(0%);
+    background: $primary-background-colour;
+  }
+}
+
+.global-nav--active {
+  transform: translateX(0%);
+}
+
+.global-nav__clumps {
+  @include transition_mobile;
+
+  position: absolute;
+  left: 0;
+  transform: translateX(0%);
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  background: $primary-background-colour;
+  font-size: 1rem;
+
+  @include respond-to($mq-l) {
+    @include transition_desktop;
+    @include column(12);
+
+    position: static;
+    background: transparent;
+    display: inline-block;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+}
+
+.global-nav__clumps--active {
+  transform: translateX(-100%);
+
+  @include respond-to($mq-l) {
+    transform: translateX(0%);
+  }
+}
+
+.global-nav__clump {
+  @include horizontal-separator;
+
+  position: static;
+  margin: 0;
+  line-height: 1;
+
+  @include respond-to($mq-l) {
+    float: left;
+    border-bottom: none;
+    border-top: none;
+    flex-grow: 1;
+  }
+}
+
+.global-nav__clump--active {
+  .global-nav__clump__content {
+    transform: translateX(100%);
+
+    @include respond-to($mq-l) {
+      display: block;
+      left: 0;
+      background: $secondary-background-colour;
+      transform: translateX(0);
+    }
+  }
+
+  .global-nav__clump__heading__text {
+    @include respond-to($mq-l) {
+      border-left: none;
+    }
+  }
+
+  @include respond-to($mq-l) {
+    background: $secondary-background-colour;
+  }
+}
+
+.global-nav__clump--active + .global-nav__clump {
+  .global-nav__clump__heading__text {
+    @include respond-to($mq-l) {
+      border-left: none;
+    }
+  }
+}
+
+.global-nav__clump:last-child {
+  border-bottom: none;
+}
+
+.global-nav__clump:first-child {
+  border-top: none;
+
+  .global-nav__clump__heading__text {
+    @include respond-to($mq-l) {
+      border-left: none;
+    }
+  }
+}
+
+.global-nav__clump__heading {
+  @include column(12);
+
+  display: inline-block;
+  float: none;
+  line-height: #{$clump-height}px;
+  padding: $baseline-unit*2 0;
+
+  @include respond-to($mq-l) {
+    display: flex;
+    width: 100%;
+    margin: 0;
+    line-height: 1;
+    text-align: center;
+
+    &:hover {
```

Comment:
Yes, I just added the hover state for a quick-win bit of functionality for demo/review purposes. In fact I think this will need to be done with JS adding the "active" class because of the structure.